definitions/features/downstream.rb

```diff
@@ -74,6 +74,14 @@ def rh_repos(sat_version)
     rh_repos
   end
 
+  def enable_ansible_repo(sat_version, rh_repos, rh_version_major)
+    if [version('6.4'), version('6.5')].include?(sat_version)
+      rh_repos << "rhel-#{rh_version_major}-server-ansible-2.6-rpms"
+    elsif sat_version == version('6.6')
```

Comment:
@upadhyeammit Will be good if we keep it  >= 6.6. This way we can avoid further changes for 6.7 as well if we keep using same ansible version.

Reply:
@ntkathole handled this condition. Marking it as fixed. Thank You !